chsdi/locale/de/LC_MESSAGES/chsdi.po

```diff
@@ -1442,18 +1442,288 @@ msgstr "Zählstelle"
 msgid "ch.astra.unfaelle-personenschaeden_alle"
 msgstr "Unfälle mit Personenschaden"
 
+msgid "ch.astra.unfaelle-personenschaeden_alle.accidentday_de"
+msgstr "Unfalltag"
+
+msgid "ch.astra.unfaelle-personenschaeden_alle.accidentday_fr"
+msgstr "Unfalltag"
+
+msgid "ch.astra.unfaelle-personenschaeden_alle.accidentday_it"
+msgstr "Unfalltag"
+
+msgid "ch.astra.unfaelle-personenschaeden_alle.accidenttype_de"
+msgstr "Unfalltyp"
+
+msgid "ch.astra.unfaelle-personenschaeden_alle.accidenttype_fr"
+msgstr "Unfalltyp"
+
+msgid "ch.astra.unfaelle-personenschaeden_alle.accidenttype_it"
+msgstr "Unfalltyp"
+
+msgid "ch.astra.unfaelle-personenschaeden_alle.accidenttypecode"
+msgstr "Code - Unfalltyp"
+
+msgid "ch.astra.unfaelle-personenschaeden_alle.accidentyear"
+msgstr "Jahr"
+
+msgid "ch.astra.unfaelle-personenschaeden_alle.canton"
+msgstr "Kanton"
+
+msgid "ch.astra.unfaelle-personenschaeden_alle.fsocommunecode"
+msgstr "BFS-Gemeinde-Nr."
+
+msgid "ch.astra.unfaelle-personenschaeden_alle.roadtype_de"
+msgstr "Strassenart"
+
+msgid "ch.astra.unfaelle-personenschaeden_alle.roadtype_fr"
+msgstr "Strassenart"
+
+msgid "ch.astra.unfaelle-personenschaeden_alle.roadtype_it"
+msgstr "Strassenart"
+
+msgid "ch.astra.unfaelle-personenschaeden_alle.roadtypecode"
+msgstr "Code - Strassenart"
+
+msgid "ch.astra.unfaelle-personenschaeden_alle.severitycategory_de"
+msgstr "Unfallschwerekategorie"
+
+msgid "ch.astra.unfaelle-personenschaeden_alle.severitycategory_fr"
+msgstr "Unfallschwerekategorie"
+
+msgid "ch.astra.unfaelle-personenschaeden_alle.severitycategory_it"
+msgstr "Unfallschwerekategorie"
+
+msgid "ch.astra.unfaelle-personenschaeden_alle.severitycategorycode"
```

Comment:
Those translations are replacing existing translations. Please remove the old translations (see mako templates)